app/lib/l10n/app_en.arb

```diff
@@ -188,5 +188,7 @@
       "placeholders": {
       "pattern": {}
       }
-  }
+  },
+  "app_outdated_message": "Your app is outdated.\nPlease update to the latest version.",
```

Comment:
A new version of the StudyU app is available. Please update it to receive the latest features and improvements. Thank you for your support!

Reply:
Fixed in https://github.com/hpi-studyu/studyu/pull/576/commits/8d36dc78771d748483bbf3ad22794a95e73dad06